gpkit/__init__.py

```diff
@@ -24,7 +24,23 @@
 
 # global variable initializations
 DimensionalityError = ValueError
-units = None
+ureg, units = None, None  # pylint: disable=invalid-name
+
+
+class GPkitUnits(object):
```

Comment:
I don't think we want to encourage pint super users to think of this as a ureg...